aptos-move/framework/aptos-framework/sources/transaction_validation.move

```diff
@@ -79,6 +79,30 @@ module aptos_framework::transaction_validation {
         txn_max_gas_units: u64,
         txn_expiration_time: u64,
         chain_id: u8,
+    ) {
+        prologue_common_v2(
+            sender,
+            gas_payer,
+            txn_sequence_number,
+            txn_authentication_key,
+            txn_gas_price,
+            txn_max_gas_units,
+            txn_expiration_time,
+            chain_id,
+            false,
+        )
+    }
+
+    fun prologue_common_v2(
+        sender: signer,
+        gas_payer: address,
+        txn_sequence_number: u64,
+        txn_authentication_key: vector<u8>,
+        txn_gas_price: u64,
+        txn_max_gas_units: u64,
+        txn_expiration_time: u64,
+        chain_id: u8,
+        has_randomness_annotation: bool,
```

Comment:
Should we call this more generically, like `deposit_gas`, etc.?

Reply:
yes!